xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodec.h

```diff
@@ -68,7 +68,9 @@ struct VideoPicture
   bool hasLightMetadata = false;
   AVContentLightMetadata lightMetadata;
 
-  AVPixelFormat pixelFormat; //< source pixel format
+  bool isHdr{false}; //< true if source is HDR (hdrType != HDR_TYPE_NONE)
+  StreamHdrType hdrType{StreamHdrType::HDR_TYPE_NONE}; //< HDR type enum
```

Comment:
that one is not necessary I think and the different values are not useful for rendering because they are ambiguous and cannot be converted back to transfer function and/or metadata type. Might as well not propagate it unless a real need exists